### ISSUE TITLE
Menu link colors

### ISSUE THREAD

**Issue by anormand** [unimelb/The-University-of-Melbourne-Web-Templates#14]
### Problem:

The menu heading color (#6685A4), i.e. ‘Example Navigation’, and menu link color (#3E658E), i.e. ‘About Us’, are very similar.
### WCAG Guideline

Success Criteria 1.4.1 - Use of Color

Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. If color alone is used to distinguish a link from surround text, it has additional differentiation (e.g. it becomes underlined) which appears when the link is hovered over or receives focus.
### Remedy:

Make it very clear what is a heading and what is a link. Using blue for both headings and links is confusing.

**Comment by ptagell** [comment]
Hi Andrew, we've made some slight tweaks to the colours. Can you let me know if this passes? We're in a bit of a bind in that we don't want this to be a completely separate color as it will either a) detract from the brand or b) draw the readers eye to the right of the page, when it should be on the content. Thoughts? http://d.pr/i5vS

**Comment by anormand** [comment]
I would take a look at the BBC web site. They have a similar color scheme to us and have used some neutral greys to differentiate between what is link text and what is ordinary text:
http://www.bbc.co.uk/news/world/africa/

**Comment by ptagell** [comment]
Take your point, but won't grey be also similar to the body text? which is a dark grey?

**Comment by ptagell** [comment]
Here's an example of the page with grey links - http://d.pr/Egqv

**Comment by anormand** [comment]
I think that you misunderstood me.

My suggestion is to use grey as the non-link color and blue as the link color.

Example (complete.css line 632):

Change
`/*  Typography  */
h1,
.nav h2,
.aside h2,
.main > h2,
#main-content > h2 {
  font-size: 16px;
  color: #56738f;
  font-family: 'Droid Sans', 'Helvetica Neue', Arial, sans-serif; }`

to
`/*  Typography  */
h1,
.nav h2,
.aside h2,
.main > h2,
#main-content > h2 {
  font-size: 16px;
  color: #777777;
  font-family: 'Droid Sans', 'Helvetica Neue', Arial, sans-serif; }`

**Comment by ptagell** [comment]
Ok - think this is now done. Sending through an edge package for you now AN.
